recipes-core/aziot-edged/aziot-edged.inc

```diff
@@ -1,13 +1,14 @@
 DEPENDS += "openssl virtual/docker iotedge aziotd systemd"
-RDEPENDS_${PN} += "docker iotedge aziotd aziot-keys systemd glibc-utils"
+RDEPENDS:${PN} += "docker iotedge aziotd aziot-keys systemd"
```

Comment:
The glibc-utils being removed here was added as part of the fix for https://github.com/Azure/meta-iotedge/issues/105 (see https://github.com/Azure/meta-iotedge/pull/109).  It looks like other changes added as part of that previous fix are still present (i.e. the pkg_postinst_ontarget_${PN} so I'd assume glibc-utils will still be needed as a runtime dependency, no?

Reply:
Thats a good point. The ownerships are set here https://github.com/Azure/meta-iotedge/blob/5513d99f97d31455bb8894deb464068945dfd74d/recipes-core/aziot-edged/aziot-edged.inc#L39-L41 so I think pkg_postint_ontarget may be not needed. Is that right @timesys-nathan ?

Reply:
@micahl @ggjjj 

My system is actually using a read only filesystem, so postinst steps are never added/executed.  I wouldn't have seen this as an issue.

I'm setting file/executable permissions in do_install() in most places, i.e.:
```
    install -d -m 755 -o iotedge -g iotedge ${D}${localstatedir}/lib/aziot/edged
```

It looks like the postinst step was to add admin and sudo users to the iotedge group.  I didn't need to do that, as I was letting systemd start the services as their respective users during initialization.  I could see wanting/needing to start/stop/resume services from an admin/sudo user directly though, so it could still be useful for some people.

I've added glib-utils back in for instances where you do want the postinst step.

Reply:
thanks for the clarification.

Reply:
Change looks good to me (will approve shortly).  @timesys-nathan, given you're also using a read only filesystem you didn't hit any issues like those reported by @sindelar-fr in https://github.com/Azure/meta-iotedge/issues/121?